### PR TITLE
Made updates to several of the newsletter sunset pages

### PR DESCRIPTION
**Pull request by ksigler7**
Description of the change: updates to several of the newsletter sunset pages
Reason for the change: requested by Molly
Link to original source: https://sendgrid.com/docs/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/index.html

### CODE REVIEW

**Comment by ksigler7** [comment]
@Whatthefoxsays can you go ahead and start reviewing the content of my changes? When I get in, I'll show you how to test the build locally, and I also need to ask Molly about a link (which is why I have it marked as status: not ready). Thanks!

File: source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/Side_by_Side_Comparisons/pricing.html [hunk 1]
```diff
@@ -31,7 +31,13 @@
       + $0.25 / 1,000 emails sent
     </td>
     <td>
-      Base Package</br>
+      Base Package plus</br>
+      Per email you send</br>
+      + $0.25 / 1,000 emails sent</br>
+      </br>
+      *OR*</br>
```
Comment (Whatthefoxsays) [inline source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/Side_by_Side_Comparisons/pricing.html]: Not rendering as bold on the website. :(

File: source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/faq.md [hunk 2]
```diff
@@ -105,37 +105,31 @@ Both Legacy Newsletter and Marketing Campaigns start with your base SendGrid pla
...
+ *Standard Marketing Campaigns pricing*</br>
+  </br>
+ $10/10,000 contacts</br>
+ (plus base plan)</td>
+   </tr>
+ </table>
 
 {% anchor h3 %}
-How can I estimate what it will cost to use Marketing Campaigns?
+How do I choose how to pay for Marketing Campaigns?
```
Comment (Whatthefoxsays) [inline source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/faq.md]: Fix the link at the top of the page too.

File: source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/faq.md [hunk 3]
```diff
@@ -105,37 +105,31 @@ Both Legacy Newsletter and Marketing Campaigns start with your base SendGrid pla
...
 {% endanchor %}
 
-Leverage our [Pricing Calculator](https://sendgrid.com/pricing/) to enter the volume of email you’ll be sending as well as the number of contacts you’ll store in Marketing Campaigns.
+You choose on the the Plan & Billing Details page in your account.
```
Comment (Whatthefoxsays) [inline source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/faq.md]: Delete "on" in "you choose on".

File: source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/index.html [hunk 4]
```diff
@@ -84,14 +84,26 @@
...
+ </br>
+$0.25/1,000 emails</br>
+(plus base plan)</td>
+        <td>Pay **per contact** you store</br>
+*Standard Marketing Campaigns pricing*</br>
+ </br>
+$10/10,000 contacts</br>
+(plus base plan)</td>
+  </tr>
+</table>
```
Comment (Whatthefoxsays) [inline source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/index.html]: Need more space between the bottom of the table and the buttons at the bottom of the page.

File: source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/index.html [hunk 5]
```diff
@@ -84,14 +84,26 @@
...
-{% endwarning %}
+You can continue to pay per email sent or you can choose to pay per contact you store in Marketing Campaigns:
+<table>
+  <tr>
+    <td>Pay **per email** you send</br>
+*Special offer for Legacy Newsletter migrators*</br>
+ </br>
+$0.25/1,000 emails</br>
+(plus base plan)</td>
+        <td>Pay **per contact** you store</br>
```
Comment (Whatthefoxsays) [inline source/User_Guide/Legacy_Newsletter/Legacy_Newsletter_Migration/index.html]: Not rendering as bold, spacing is wonky.

**Pull request merged**